Makefile

```diff
@@ -12,7 +12,7 @@ install:
 	pip install .
 
 install-hooks:
-	tox -e pre-commit -- install -f --install-hooks
+	tox -e pre-commit
```

Comment:
for my understanding - what is the reason for removing "-- install -f --install-hooks"?

Reply:
The `pre-commit` env in the `tox.ini` passes additional positional arguments to `pre-commit run --all-files`. I think this target was written before a change in that setup when this made more sense.